components/form/form-element-mixin.js

```diff
@@ -77,21 +77,34 @@ export class FormElementValidityState {
 export const FormElementMixin = superclass => class extends LocalizeCoreElement(superclass) {
 
 	static get properties() {
-		return { validationError: { attribute: false, type: String } };
+		return {
+			forceInvalid: { type: Boolean, attribute: false },
+			invalid: { type: Boolean, reflect: true },
+			validationError: { type: String, attribute: false },
+		};
 	}
 
 	constructor() {
 		super();
 		this._validationCustoms = new Set();
 		this._validationMessage = '';
 		this._validity = new FormElementValidityState({});
+		this.forceInvalid = false;
```

Comment:
Does `invalid` need to be defaulted to `false`, or does setting `forceInvalid` here cause the `updated()` call to set it?

Reply:
Yeah, I didn't initialize it because it should get set to its value when the component does its first `updated` call. Since its effectively a computed property I only wanted its value to be set in `updated()`.

Reply:
Not sure if not initializing it in the constructor will mess up @margaree's JSDoc stuff. Alternately, we might need to add `@default false` to the JSDoc comments on the property.

Reply:
If it makes other things more difficult we might as well initialize it, it shouldn't matter if its `undefined` or `false` before the first `updated()` call.